Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py

```diff
@@ -204,7 +218,7 @@ def execute_query_command(client: Client, args: dict[str, Any]) -> CommandResult
     timeout = arg_to_number(args.get('timeout', 10))
     workspace_id = args.get('workspace_id') or demisto.params().get('workspaceID')
 
-    full_url = f'https://api.loganalytics.io/v1/workspaces/{workspace_id}/query'
+    full_url = f"{client.azure_cloud.endpoints.log_analytics_resource_id}/v1/workspaces/{workspace_id}/query"
```

Comment:
This shouldnt be depending on the full_url argument as well?

Reply:
Not sure I understand.
I didn't change the logic here, just added a dynamic endpoint.
This is not effecting the original logic and was tested with both "regular" and "GCC High" credentials.

Reply:
In resource_group_list_request we had default, why not here?

Reply:
As far as I understand, this should be the only URL when executing a query using the `azure-log-analytics-execute-query` command. I'm not aware of any other default url for this use-case.